edx/edx-enterprise
# [ENT-349] Display Course Info Overlay

**Description:** This PR displays a modal dialog when the "View Course Details" link is clicked in the landing page.

**JIRA:** [ENT-349](https://openedx.atlassian.net/browse/ENT-349)

**Dependencies:** None. 

**Installation instructions:** Standard Enterprise setup

**Screenshot:**
![screenshot](https://user-images.githubusercontent.com/560781/26891889-d0839cee-4b73-11e7-805c-8e2c939346f5.png)


**Testing instructions:**

1. Add an Enterprise Customer in the Django admin. Check its ID
2. Add a course in the customer catalog. Check its ID.
3. Visit the new landing page for the enterprise customer and course by typing a URL like this:

http://localhost:8000/enterprise/7c3c286a-9834-46ae-809e-e01b114ba980/course/course-v1:edX+DemoX+Demo_Course/enroll/

where 7c3c286a-9834-46ae-809e-e01b114ba980 should be the ID of the customer and 'course-v1:edX+DemoX+Demo_Course' should be the ID of the course

4. Click the "View Course Details" in the landing page
5. Verify that a modal dialog like the screenshot above opens
6. Click the close button or outside the dialog to close it 

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.~

**Post merge:**

**Author concerns:**

1. The ENT ticket mentions several attributes to include, but I followed the image mockup as much as possible. Also the following information in the dialog has been mocked because I didn't know where to find it: 
- course level
- course length
- the what you'll learn section at the bottom

2. Also the ENT mockup has a "You Save $10.00" message in the price that I cannot generate because the original and final price attributes are just strings. So I omitted that for the moment. And for displaying the price I just got the first course mode.

**Reviewers**
